web/modules/custom/copyright/src/Plugin/Block/Copyright.php

```diff
@@ -55,10 +55,21 @@ public static function create(
    * {@inheritdoc}
    */
   public function build() {
-    $storage = $this->configPagesLoaderService->load('global_configurations');
-    $copyrightField = $storage->get('field_copyright')->view('default');
-    $copyrightField['#title'] = '';
-    $copyrightField['#cache']['tags'][] = 'config_pages:1';
+    $loader = $this->configPagesLoaderService;
+    if ($storage = $loader->load('global_configurations')) {
```

Comment:
this is not a storage, but an entity.
Storage in Drupal kind of refers to instance of entity type storage in most cases, so this can be misleading